website/addons/base/views.py

```diff
@@ -80,6 +80,7 @@ def check_file_guid(guid):
     'delete': 'write',
     'copy': 'write',
     'move': 'write',
+    'rename': 'write',
```

Comment:
These fields are used by waterbutler, rename would not be used here. Please remove